### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by suerfang** [lncrawl/lightnovel-crawler#1887] [Ideas]
I am from another country and I wanted to see if it is possible to add another page to the project. This is "es.mtlnovel.com" and "wuxiaworld.site" are pages where you can read novels in Spanish, the program you made is fantastic, but there is no novel in Spanish. Additionally, it seems there is an error with the formatters because when I try to download in pdf format, it doesn't download in that format.

**Comment by dipu-bd** [comment]
It is possible. Just open a Source Request issue here: https://github.com/dipu-bd/lightnovel-crawler/issues/new?assignees=&labels=source&template=2_new-source.md&title=Replace+this+with+an+url
